galaxy_ng/app/tasks/synclist.py

```diff
@@ -20,7 +19,8 @@
 from galaxy_ng.app import models
 
 log = logging.getLogger(__name__)
-pf = pprint.pformat
+
+SYNCLIST_BATCH_SIZE = 200
```

Comment:
should this be configurable in settings.py? That way we can tweak it without a code deployment